src/bin/commands/new-project/index.ts

```diff
@@ -156,11 +170,13 @@ export const newCdkProject = async (props: NewProjectProps): CliCommandResponse
 
   console.log("Project successfully created! Next steps:");
```

Comment:
I know that this PR isn't updating the logic, but I think the next steps here should only be logged for migrations i.e. `if (config.yamlTemplateLocation)`?

Reply:
I think we can simplify this even further by simply saying "Project successfully created! For next steps, see the docs", as it keeps the code super simple, and allows us to update the next steps w/out a new release.

Reply:
Will make these changes on this PR.

Reply:
@jacobwinch I've added a new commit for this now.